docs/supported-services/route53zone.rst

```diff
@@ -0,0 +1,174 @@
+.. _route53zone:
+
+Route 53 Hosted Zone
+====================
+This document contains information about the Route 53 Hosted Zone service supported in Handel. This Handel service provisions a Route 53 Hosted Zone, in which you can create other DNS records.
+
+Service Limitations
+-------------------
+The following Route 53 features are not currently supported in this service:
+
+* Domain Name Registration
+
+Manual Steps
+------------
+If creating a public zone as a subdomain of another domain (like myapp.byu.edu), you must register it with your DNS provider.
+
+For BYU users, please refer to `this document <https://byuoit.atlassian.net/wiki/spaces/OAPP/pages/40075276/Routing+BYU+DNS+into+AWS>`_.
+
+Parameters
+----------
+.. list-table::
+   :header-rows: 1
+
+   * - Parameter
+     - Type
+     - Required
+     - Default
+     - Description
+   * - type
+     - string
+     - Yes
+     -
+     - This must always be *route53zone* for this service type.
+   * - name
+     - string
+     - Yes
+     -
+     - The DNS name for this hosted zone.
+   * - private
+     - boolean
+     - No
+     - false
+     - Whether or not this is a private zone. If it is a private zone, it is only accessible by the VPC in your account config file.
+   * - tags
+     - :ref:`route53zone-tags`
+     - No
+     -
+     - Any tags you want to apply to your Hosted Zone
+
+.. _route53zone-tags:
+
+Tags
+~~~~
+The Tags element is defined by the following schema:
+
+.. code-block:: yaml
+
+  tags:
+   <your_tag_name>: <your_tag_value>
+
+.. NOTE::
+
+    Handel automatically applies some tags for you. See :ref:`tagging-default-tags` for information about these tags.
+
+Example Handel File
+-------------------
+
+.. code-block:: yaml
+
+    version: 1
+
+    name: my-dns
+
+    environments:
+      dev:
+        public-zone:
+          type: route53zone
+          name: mydomain.byu.edu
+          tags:
+            mytag: mytagvalue
+        private-zone:
+          type: route53zone
+          name: private.myapp # Doesn't have to be a .byu.edu domain
+          private: true
+          tags:
+            mytag: mytagvalue
+
+Depending on this service
```

Comment:
Yeah, the use case I thought of is if you need to know what your public-facing URL is (like to do CAS redirects), you can avoid hardcoding it by just depending on your hosted zone.